src/components/PricingTable.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+  import PricingTableCard from "./PricingTableCard.svelte";
+
+  const pricingCards = [
+    {
+      title: "Free",
+      description:
+        "Use all features of Authgear to boost your app's security and develop faster",
+      features: [
+        "1,000 Monthly Active Users",
+        "1 Day of Log Retention",
+        "Account remove if no activities for 60 days",
+        "No SLA",
+        "Community Support",
+      ],
+    },
+    {
+      title: "Startups",
+      description: "Grow without pain and security",
+      features: [
+        "1,000 Monthly Active Users",
+        "Up to 10 Days of Log Retention",
+        "AD, LDAP, SAML support",
+        "Passwordless Login / Verify by SMS",
+        "Email Support",
+      ],
+      pricingText: "USD 200",
+      pricingSubtext: "+USD 20 per 1000 Monthly Active Users",
+    },
+    {
+      title: "Enterprise",
+      description: "Run Authgear for your mission-critical applications",
+      features: [
+        "Managed SaaS or Private Cloud Deployment",
+        "Enterprise Grade SLA support",
+        "Professional Support and Services",
+        "Anomaly Detection",
+        "Audited Reports",
+      ],
+      buttonText: "Pricing: Contact Us",
+      buttonHref: "mailto:hello@skygear.io",
+    },
+  ];
+  let colSize = Math.floor(12 / pricingCards.length);
```

Comment:
For cards, we usually want a fixed size for each screen size, instead of dynamic size depending on number of cards.

Therefore, I'll suggest:
- Hard-code the size of cards directly in class names.
- Center the cards in the rows.